tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequireServer.cs

```diff
@@ -162,6 +162,18 @@ public RequireServer StorageEngines(params string[] storageEngines)
             throw new SkipException($"Test skipped because storage engine is \"{actualStorageEngine}\" and not one of ({storageEnginesString}).");
         }
 
+        public RequireServer Tls(bool required = true)
+        {
+            var usingTls = CoreTestConfiguration.ConnectionString.Tls;
+            if (CoreTestConfiguration.ConnectionString.Tls == required)
```

Comment:
This could be shortened a bit:

     if (usingTls == required)

Reply:
Ah true, done!